### PR TITLE
Tag NPZ v0.2.0 [https://github.com/fhs/NPZ.jl]

### PR DESCRIPTION
**Pull request by fhs**
Fixes fhs/NPZ.jl#15

### CODE REVIEW

File: NPZ/versions/0.2.0/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+julia 0.4
+ZipFile
+Compat
```
Comment (tkelman) [inline NPZ/versions/0.2.0/requires]: It's best to declare a minimum version dependency that implements the features you need.

**Comment by fhs** [comment]
I've added versions to requires file.

File: NPZ/versions/0.2.0/requires [hunk 2]
```diff
@@ -0,0 +1,3 @@
+julia 0.4
+ZipFile 0.3.0
+Compat 0.8.4
```
Comment (tkelman) [inline NPZ/versions/0.2.0/requires]: are you using `Compat.view` ? `Compat.String` was 0.7.15

Comment (fhs) [inline NPZ/versions/0.2.0/requires]: I don't think so. However, one of the dependency (ZipFile) requires Compat 0.8.4

Comment (tkelman) [inline NPZ/versions/0.2.0/requires]: there could conceivably be future versions of ZipFile where that is no longer the case though

Comment (fhs) [inline NPZ/versions/0.2.0/requires]: I've changed Compat version to 0.7.15

**Pull request merged**